specs/AgriFood/AgriGreenhouse/schema.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/FIWARE/dataModels/agriDataModels/specs/AgriFood/AgriGreenhouse/schema.json",
```

Comment:
you don't need such a long URL, please check the other FIWARE Data Models.

This comment applies for the rest of PRs